charms/worker/k8s/src/charm.py

```diff
@@ -79,7 +81,7 @@ def _apply_snap_requirements(self):
         """
         status.add(ops.MaintenanceStatus("Applying K8s requirements"))
         commands = [
-            "/snap/k8s/current/k8s/connect-interfaces.sh",
+            #"/snap/k8s/current/k8s/connect-interfaces.sh",
```

Comment:
This fails for classic snaps so I removed it for now. We should probably just adapt the script in the future.